### DISCUSSION TITLE
Ikea TRADFRI motion sensor binded to group but not switch IKEA lights on

### DISCUSSION THREAD

**Discussion by ampujl** [Koenkk/zigbee2mqtt#16713] [Devices]
Hi,
I have Ikea motion sensor and binded it to group containing 2 Ikea light bulbs. 
When lights are turned off and motion sensor occupancy turn true, nothing happens. 
What i do wrong?

![image](https://user-images.githubusercontent.com/57711934/219640176-de21e654-1228-44f2-bced-ce0148c7641a.png)

I also created 2 scenes in z2m: one lights turned on and one lights turned off:
- recall on => lights turning on
- recall off => lights state in group view turns off but lights stays bright

**Comment by Revyen** [comment]
Tried the same thing on 3 diffrent ikea motion sensors with 3 diffrent groups to no avail.

I got it working for some time, after I accidentally deleted my groups I could't get it to work again for some reason.

**Comment by malaakso** [comment]
Could be related to #7716 ? Have you tried to bind to individual lights? Also, have you verified that the illuminance sensor is disabled with the button at the back of the sensor, otherwise the lights turn on only when it is dark enough.
